#!/usr/bin/env python

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()
#       Name              Type          level               Description       Default     Min         Max
gen.add("linear_x",       double_t,     0,                  "linear X",       0.0,       -10.0,       10.0)
gen.add("angular_z",      double_t,     0,                  "angular Z",      0.0,       -10.0,       10.0)

exit(gen.generate("chapter5", "dynamic_param_node", "DynamicParam"))

